src/Resources/app/administration/src/init/flag-components.init.js

```diff
@@ -1,5 +0,0 @@
-import flagComponents from '../assets';
-
-flagComponents.forEach((flagComponent) => {
-    Shopware.Component.register(flagComponent.name, flagComponent);
```

Comment:
Could use a proper deprecation, just in case.

Maybe log a warning in the mounted/created hook of this component